terraform/variables.tf

```diff
@@ -0,0 +1,159 @@
+variable "model" {
+  type    = string
+  default = "jimm"
+}
+
+variable "include_oauth-external-idp-integrator" {
+  description = <<EOT
+  If you set this to true, you need to configure the oauth-external-idp-integrator charm.
+  In this way you can use the JIMM charm without the iam-bundle.
+  EOT
+  type        = bool
+  default     = false
+}
+
+variable "name" {
+  description = "JIMM application name"
+  type        = string
+  default     = "jimm"
+}
+
+variable "trust" {
+  description = "The status to grant the Juju application full access to the cluster."
+  type        = bool
+  default     = true
+}
+
+
+variable "charm" {
+  description = "The application charm operator information."
+  type = object({
+    name : string
+    channel : string
+    base : string
+  })
+  default = {
+    name    = "juju-jimm-k8s"
+    channel = "3/edge"
```

Comment:
can you please add a revision, too.. so that ppl can fix a revision of jimm they want to deploy?